packages/make-styles/package.json

```
@@ -37,7 +37,7 @@
     "csstype": "^2.6.7",
     "inline-style-expand-shorthand": "^1.2.0",
     "rtl-css-js": "^1.14.0",
-    "stylis": "^4.0.6",
+    "stylis": "4.0.6",
```

```suggestion
    "stylis": "^4.0.6",
```

This is a production (currently) dependency on consumers' side, but anyway we can remove a caret temporary there if it breaks something.

However, I don't think that we should restrict version ranges without an explicit need as this can cause issues for bundle size via duplication of dependencies that are potentially in semver range.